test/kitchen-sink/main.ts

```diff
@@ -216,5 +222,21 @@ import { getReferrer } from "../../src/device"; // Private API!
             updateLeaderboardUI();
         }
     };
+
+
+    document.querySelector("#subscribeSms").onclick = async () => {
+        await playpass.chatbots.subscribeToSms("do you consent");
```

Comment:
Where is this consent text used?

Reply:
This is just stored alongside the user, I made an assumption here about legals requirement's based on them already requiring us to store the sms signup text. 

This should be the body of the text you use for the `do you consent to receive notifcations with sms`?